### PR TITLE
Change e2e test context name

### PR DESCRIPTION
**Pull request by paulfantom**
cc: @ingvagabund

### CODE REVIEW

File: ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-presubmits.yaml [hunk 1]
```diff
@@ -32,6 +32,6 @@ presubmits:
     agent: jenkins
     labels:
       master: ci.openshift.redhat.com
-    context: ci/openshift-jenkins
+    context: ci/jenkins/e2e
```
Comment (ingvagabund) [inline ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-presubmits.yaml]: Let's do `ci/jenkins/mao-e2e`

Comment (paulfantom) [reply ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-presubmits.yaml]: that name will only show up in mao repo, so why add `mao`?

Comment (ingvagabund) [reply ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-presubmits.yaml]: Later one we can have more e2e tests. Let's make clear it's for the mao.

Comment (paulfantom) [reply ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-presubmits.yaml]: Don't get it. Later we might have more e2e tests in mao repository which won't include mao? Strange.

Comment (ingvagabund) [reply ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-presubmits.yaml]: let's do it later then

**Pull request merged**